examples/jsm/renderers/nodes/accessors/CameraNode.js

```diff
@@ -0,0 +1,96 @@
+import Node from '../core/Node.js';
+import Vector3Node from '../inputs/Vector3Node.js';
+import Matrix4Node from '../inputs/Matrix4Node.js';
+import { NodeUpdateType } from '../core/constants.js';
+
+class CameraNode extends Node {
+
+	static POSITION = 'position';
+	static PROJECTION = 'projection';
+	static VIEW = 'view';
+
+	constructor( scope = CameraNode.POSITION ) {
+
+		super();
+
+		this.updateType = NodeUpdateType.Frame;
+
+		this.scope = scope;
+
+		this._inputNode = null;
+
+	}
+
+	getType() {
+
+		const scope = this.scope;
+
+		if ( scope === CameraNode.PROJECTION || scope === CameraNode.VIEW ) {
+
+			return 'mat4';
+
+		}
+
+		return 'vec3';
+
+	}
+
+	update( frame ) {
+
+		const camera = frame.camera;
+		const inputNode = this._inputNode;
+		const scope = this.scope;
+
+		if ( scope === CameraNode.PROJECTION ) {
+
+			inputNode.value = camera.projectionMatrix;
+
+		} else if ( scope === CameraNode.VIEW ) {
+
+			inputNode.value = camera.matrixWorldInverse;
+
+		} else if ( scope === CameraNode.POSITION ) {
+
+			camera.getWorldPosition( inputNode.value );
+
+		}
+
+	}
+
+	generate( builder, output ) {
+
+		const nodeData = builder.getDataFromNode( this );
+
+		let inputNode = this._inputNode;
+
+		if ( nodeData.inputNode === undefined ) {
+
+			const scope = this.scope;
+
+			if ( scope === CameraNode.PROJECTION || scope === CameraNode.VIEW ) {
+
+				if ( !inputNode || !inputNode.isMatrix4Node ) {
```

Comment:
Please this code style:
```js
if ( inputNode === undefined || inputNode.isMatrix4Node !== true ) {
```
We compare `is*` properties always against `true`.

Reply:
Yeah! I miss, thanks for remember me.